packages/financial-templates-lib/test/clients/OptimisticOracleClient.js

```diff
@@ -113,23 +118,23 @@ contract("OptimisticOracleClient.js", function (accounts) {
 
     // Initially, no proposals and no price requests.
     let result = client.getUndisputedProposals();
-    assert.deepStrictEqual(result, []);
+    objectsInArrayInclude(result, []);
     result = client.getUnproposedPriceRequests();
-    assert.deepStrictEqual(result, []);
+    objectsInArrayInclude(result, []);
     result = client.getSettleableProposals(proposer);
-    assert.deepStrictEqual(result, []);
+    objectsInArrayInclude(result, []);
 
     // Request and update again, should show no proposals.
     await optimisticRequester.requestPrice(identifier, requestTime, "0x", collateral.address, 0);
     await client.update();
     result = client.getUndisputedProposals();
-    assert.deepStrictEqual(result, []);
+    objectsInArrayInclude(result, []);
     result = client.getSettleableProposals(proposer);
-    assert.deepStrictEqual(result, []);
+    objectsInArrayInclude(result, []);
 
     // Should have one price request.
     result = client.getUnproposedPriceRequests();
-    assert.deepStrictEqual(result, [
+    objectsInArrayInclude(result, [
```

Comment:
it seems like the parameters are reversed from the other comment

Reply:
Yeah, different tests had used different orderings of these values. That's why the order of the args is swapped in one file. I can go through and correct this -- I just wanted to avoid deeper modifications of these tests.

Thoughts?

Reply:
np just wantd to call it out incase it was not intended